docs/README.md

```diff
@@ -0,0 +1,10 @@
+# ssis.bit
+AIO solution for SSIS students
+
+This is just a placeholder. You might be looking for these files:
+
+[index.md](index.md) The file for the website [ssisbit.github.io/ssis.bit](https://ssisbit.github.io/ssis.bit)
+
+[history.md](history.md) A little history of this project.
+
```

Comment:
Thanks for inlcuding the /docs structure. But be aware: at least /README.md and /docs/index.md have to be updated regularly.

What about a webpage https://ssisbit.org/ like for the micro:bit ?